commercial/app/controllers/commercial/Multi.scala

```diff
@@ -75,15 +75,15 @@ object Multi
               }
             case ("masterclasses", eventBriteId) if eventBriteId.nonEmpty =>
               Future.successful {
-                MasterclassAgent.specificMasterclasses(Seq(eventBriteId)).headOption orElse {
-                  MasterclassAgent.masterclassesTargetedAt(segment).headOption
+                MasterclassAgent.specificMasterclasses(Seq(eventBriteId)).filter(!_.mainPicture.isEmpty).headOption orElse {
+                  MasterclassAgent.masterclassesTargetedAt(segment).filter(!_.mainPicture.isEmpty).headOption
```

Comment:
More of an FYI than a suggestion to make a change but there is a `filterNot` method that can be used rather than negating an expression within a `filter` method.